rest-api-spec/src/main/resources/rest-api-spec/test/cluster.master/10_basic.yml

```diff
@@ -0,0 +1,10 @@
+---
+"get cluster master":
+  - skip:
+      version: " - 7.99.99"
+      reason: Get cluster master API was introduced in 8.0
```

Comment:
After backporting, as a follow up, this `skip` can be removed, and all the REST tests that get the master ID from the cluster state can migrate to using this API instead.